template/nuxt/nuxt.config.js

```diff
@@ -159,6 +162,25 @@ module.exports = {
     }
   },
   <%_ } _%>
+  <%_ if (purgecss) { _%>
+  /*
+  ** PurgeCSS module configuration
+  ** https://github.com/Developmint/nuxt-purgecss
+  */
+  purgeCSS: {
+    mode: 'postcss',
+    paths: [
+      'components/**/*.vue',
+      'layouts/**/*.vue',
+      'pages/**/*.vue',
+      'plugins/**/*.js',
+      './node_modules/vuetify/src/**/*.ts'
```

Comment:
Is  this line included only when ui is vuetify?

Reply:
I’ll try to do that and request a new pull.

Reply:
Addressed this in an [update](https://github.com/nuxt/create-nuxt-app/pull/430/commits/4a5edfc6366015ea357b6a09143ea7051f3f83c9)